planner/core/explain.go

```diff
@@ -890,11 +891,23 @@ func (p *LogicalAggregation) ExplainInfo() string {
 	return buffer.String()
 }
 
+func (p *LogicalAggregation) buildLogicalPlanTrace() *tracing.LogicalPlanTrace {
```

Comment:
Are we going to have different implementations for different logical plans? If not, can we align them to one func?

Reply:
We need to attach explain info in trace, thus it should be implemented by each logical operator.

Reply:
ExplainInfo is also defined as an interface, what about "func buildLogicalPlanTrace(p *baseLogicalPlan) {...}"

Reply:
updated.